t/test.pl

```diff
@@ -1733,8 +1737,16 @@ ($;$)
     my $method  = shift || "";
     my $timeout_msg = 'Test process timed out - terminating';
 
+    # Accept either spelling
+    my $timeout_factor = $ENV{PERL_TEST_TIME_OUT_FACTOR}
```

Comment:
FWIW, I was thinking of taking this a teeny step further.
```
my $script= $0;
$script=~s!/!_!g;
my $timeout_factor = $ENV{"PERL_TTF_$script"} || $ENV{PERL_TEST_TIME_OUT_FACTOR}
```

That is where we can set an environment variable to make a specific test script do this. So we have two level of settings "give all my tests extra time" and "give this script extra time".

Reply:
For the record, in case you feel inclined to change it further, I still think it would be nice to be able to turn the watchdog off altogether. E.g.  make an immediate return if the 'timeout' resolves to '0' or negative or so.